packages/next-swc/crates/emotion/src/lib.rs

```diff
@@ -337,7 +337,21 @@ impl<C: Comments> EmotionTransformer<C> {
             if index % 2 == 0 {
                 if let Some(q) = tagged_tpl.quasis.get_mut(i) {
                     let q = q.take();
-                    let minified = minify_css_string(&q.raw, index == 0, index == args_len - 1);
+                    let input = match q.cooked {
+                        Some(cooked) => cooked,
+                        _ => {
+                            HANDLER.with(|handler| {
+                                handler
+                                    .struct_span_err(
```

Comment:
I wanted to make this a warning, but `.struct_span_warn()` does not cause any output (at least when running `next dev`). Is there a reason for that?

Reply:
I misdesigned the API 🤣